Cargo.toml

```diff
@@ -11,6 +11,7 @@ members = [
     "substreams-helper",
     "uniswap-v2",
     "solana-sample",
+    "ens",
```

Comment:
I'd rename this as `ens-lookup` as ENS can mean a lot of things (e.g. could be a substream for ENS the protocol)